src/ucp/core/ucp_worker.inl

```diff
@@ -8,30 +8,39 @@
 #define UCP_WORKER_INL_
 
 #include "ucp_worker.h"
+#include "ucp_rkey.inl"
 
 #include <ucp/core/ucp_request.h>
 #include <ucp/wireup/address.h>
 #include <ucs/datastruct/ptr_map.inl>
 
 
-static UCS_F_ALWAYS_INLINE khint_t
-ucp_worker_rkey_config_hash_func(ucp_rkey_config_key_t rkey_config_key)
-{
-    return (khint_t)rkey_config_key.md_map ^ rkey_config_key.ep_cfg_index ^
-           (rkey_config_key.mem_type << 16);
-}
+KHASH_IMPL(ucp_worker_rkey_config, ucp_rkey_config_key_t,
+           ucp_worker_cfg_index_t, 1, ucp_rkey_config_hash_func,
+           ucp_rkey_config_is_equal);
 
-static UCS_F_ALWAYS_INLINE int
-ucp_worker_rkey_config_is_equal(ucp_rkey_config_key_t rkey_config_key1,
-                                ucp_rkey_config_key_t rkey_config_key2)
+/**
+ * Resolve remote key configuration key to a remote key configuration index
+ *
+ * @param [in]  worker          UCP worker to resolve configuration on
+ * @param [in]  key             Rkey configuration key
+ * @param [in]  lanes_distance  Distance of each lane to the remote memory
+ *                              buffer on the peer.
+ * @param [out] cfg_index_p     Filled with configuration index in the worker.
```

Comment:
minor: please either add  `.` for each line in the end, or remove the existing ones for consistency